README.md

```diff
@@ -56,8 +56,6 @@ If you don't see a version number, then make sure Git is on your `PATH`.
 
 ## Step 4: Clone this repository
```

Comment:
Line 64: another TODO that should be removed.

Reply:
Good catch. Done.